modules/react/src/components/export-video/export-video-panel-container.js

```diff
@@ -75,13 +76,40 @@ export class ExportVideoPanelContainer extends Component {
   }
 
   componentDidMount() {
+    const {
+      mapData: {
+        visState: {animationConfig, filters}
+      },
+      onTripFrameUpdate,
+      onFilterFrameUpdate
+    } = this.props;
     const animation = new KeplerAnimation({
+      filters,
+      filterKeyframes: this.getFilterKeyframes(),
+      animationConfig,
+      tripKeyframe: {timings: [0, this.state.durationMs]},
       cameraKeyframe: this.getCameraKeyframes(),
-      onCameraFrameUpdate: this.setViewState
+      onCameraFrameUpdate: this.setViewState,
+      onTripFrameUpdate,
+      onFilterFrameUpdate
     });
     this.state.adapter.animationManager.attachAnimation(animation);
   }
 
+  getFilterKeyframes() {
+    const {
+      mapData: {
+        visState: {filters}
+      }
+    } = this.props;
+    return filters
+      .filter(f => f.type === 'timeRange')
```

Comment:
Generates filter animation for all timeRange filters. We may want to instead do it for the active filter.